bin/ashley

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
-declare DOCKER_USER
-DOCKER_USER="$(id -u):$(id -g)"
+# shellcheck source=bin/_config.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
 
-DOCKER_USER=${DOCKER_USER} docker-compose up -d postgresql
-DOCKER_USER=${DOCKER_USER} docker-compose run --rm dockerize dockerize -wait tcp://postgresql:5432 -timeout 60s
-DOCKER_USER=${DOCKER_USER} docker-compose run --rm --service-ports ashley
+_docker_compose up -d postgresql
+_dc_run dockerize dockerize -wait tcp://postgresql:5432 -timeout 60s
+_dc_run --service-ports ashley
```

Comment:
Since we define port mapping in the `docker-compose` configuration, is this option required?

Reply:
Yep, apparently with `docker-compose run`, this option it is required.

`docker-compose status` (with the option):
```
         Name                        Command               State            Ports         
------------------------------------------------------------------------------------------
ashley_ashley_run_1       /usr/local/bin/entrypoint  ...   Up      0.0.0.0:8090->8000/tcp 
ashley_postgresql_1       docker-entrypoint.sh postgres    Up      0.0.0.0:15432->5432/tcp
```

`docker-compose status` (without the option) :
```
         Name                        Command               State            Ports         
------------------------------------------------------------------------------------------
ashley_ashley_run_1       /usr/local/bin/entrypoint  ...   Up                             
ashley_postgresql_1       docker-entrypoint.sh postgres    Up      0.0.0.0:15432->5432/tcp
```

Reply:
Oh I see. The alternative would be to use:

```suggestion
_docker_compose up -d ashley
```

Reply:
But it's fine as is.